README.md

```diff
@@ -292,7 +292,7 @@ can do the below:
 ```elixir
 binary_file_content = "Something you fetched and now have it in memory"
 token = "some_token_from_another_request"
-headers = ["Authorization": "Bearer #{token}", {"Content-Type", "multipart/form-data"}]
+headers = [Authorization: "Bearer #{token}", "Content-Type": "multipart/form-data"]
```

Comment:
I think the best solution here is to use tuples with strings instead of atoms :thinking: . I'm assuming this was the original intention

```elixir
[{"Authorization, "Bearer #{token}"}, {"Content-type", "multipart/form-data"}]
```

WDYT?

Reply:
Thanks a lot for your review.
Error message has a keyword list.

> unexpected expression after keyword list. Keyword lists must always come last in lists and maps. Therefore, this is not allowed:

So I assume a keyword list.

I updated to use tupples.